test/web3data.test.js

```diff
@@ -53,6 +53,11 @@ test('Web3data rawQuery accepts url and returns valid response', async t => {
     t.is(response.status, 200)
 });
 
+test('Web3data getEtherPrice returns valid response', async t => {
+    let response = await t.context.web3data.getEtherPrice();
```

Comment:
@taylorjdawson travis is still failing on this:
https://travis-ci.com/web3data/web3data-js/builds/120249016#L529